rust/utils/run-locally/src/main.rs

```diff
@@ -158,10 +158,10 @@ fn main() -> ExitCode {
         "HYP_BASE_CHAINS_TEST2_SIGNER_TYPE" => "hexKey",
         "HYP_BASE_CHAINS_TEST3_SIGNER_KEY" => "701b615bbdfb9de65240bc28bd21bbc0d996645a3dd57e7b12bc2bdf6f192c82",
         "HYP_BASE_CHAINS_TEST3_SIGNER_TYPE" => "hexKey",
-        "HYP_RELAYER_GASPAYMENTENFORCEMENT_POLICY_TYPE" => "none",
+        "HYP_RELAYER_GASPAYMENTENFORCEMENT" => r#"[{"type": "none"}]"#,
```

Comment:
we'll need to make sure that the accompanying TS infra / helm logic for this stuff is updated as well. Especially around the existing whitelisting of interchain query messages -- lmk if I can help at all with this